stats/lib/platform-stats-fetchers.js

```diff
@@ -43,6 +43,22 @@ export const fetchDailyStationAcceptedMeasurementCount = async (pgPool, filter)
   return rows
 }
 
+/**
+ * @param {Queryable} pgPool
+ * @param {import('./typings.js').DateRangeFilter} filter
+ */
+export const fetchTopMeasurementStations = async (pgPool, filter) => {
+  const { rows } = await pgPool.query(`
+    SELECT station_id, SUM(accepted_measurement_count) as total_accepted_measurement_count
+    FROM daily_stations
+    WHERE day >= $1 AND day <= $2
```

Comment:
What data do we need for the leaderboard? Do we need to aggregate historical data (e.g. over the last 7 days) or is it enough to always use the data from the previous day only? Or even today's data only?

The shorter the interval, the faster and cheaper the query will be.

Reply:
@patrickwoodhead Any opinion here?

Reply:
We agreed to use the data from the previous day only. Marking this thread as resolved.